src/API/Site/Controllers/Ads/AssetGroupController.php

```diff
@@ -55,14 +55,20 @@ public function register_routes(): void {
 			]
 		);
 		$this->register_route(
-			'ads/campaigns/(?P<id>[\d]+)/asset-groups',
+			'ads/campaigns/asset-groups',
 			[
 				[
 					'methods'             => TransportMethods::READABLE,
 					'callback'            => $this->get_asset_groups_by_campaign_id_callback(),
 					'permission_callback' => $this->get_permission_callback(),
 					'args'                => $this->get_asset_group_params(),
 				],
+				[
+					'methods'             => TransportMethods::CREATABLE,
+					'callback'            => $this->create_asset_group_callback(),
```

Comment:
💅 I personally will rename the previous `get_asset_groups_by_campaign_id_callback()` as just `get_asset_groups_callback` to maintain consistency.